images/twitter.css

```diff
@@ -0,0 +1,314 @@
+*{
+    font-family: Arial, Helvetica, sans-serif;
+}
+.bigcontainer{
+    display: grid;
+    grid-template-columns: 5fr 10fr 5fr;
+    grid-template-areas: "left middle right";
+    background-color: whitesmoke;
+    gap: 5px;
+}
+.leftcontainer{
```

Comment:
I like that this is a grid. Also the menu looks good with the icons and words being the same size, font style etc. I think the grid style will also futureproof your code for media-queries and responsiveness (? not sure how those work tbh but I think the gird is a good way to go)